src/common/MovieCardFull/index.js

```diff
@@ -0,0 +1,32 @@
+import { Title, Container, Image, Info, Subtitle, DetailBox, Production, Release, Label, TagList, Tag, Rating, StarRating, VoteCount, OutOf, Description, Star } from "./styled";
+
+const MovieCardFull = ({ description, imageURL, title, production, release, tags, rating, voteCount }) => {
+    return (
+        <Container>
+            <Image src={imageURL} />
+            <Info>
+                <Title>{title}</Title>
+                <Subtitle>2021</Subtitle>
```

Comment:
I think here should't be only string of numbers, but probably date from Api

Reply:
oops didnt see that